src/components/common/contact-form.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import ContactFormStyles from './contact-form.module.scss'
+
+export default ({ name, messageCopy }) => {
+  return (
+    <form
+      className={ContactFormStyles.contactForm}
+      name={name}
+      method="POST"
+      action="/contact/success"
+      netlify-honeypot="covid-bot-field"
+      data-netlify="true"
+    >
+      <div>
+        <input type="hidden" name="form-name" value={name} />
+        <label htmlFor="name">
+          <p className={ContactFormStyles.inputHeader}>
+            What is your email address?
+          </p>
+          <input type="email" id="name" name="name" required />
```

Comment:
Is there a way we could make the `id` / `name` attrs be equal to `email` or similar? `name` seems like a confusing label for an email field.